docs/tre-admins/setup-instructions/installing-base-workspace.md

```diff
@@ -2,14 +2,14 @@
 
 ## Publishing and registering the base workspace bundle
 
-Run the following in a terminal: -
+Run the following in a terminal:
 
   ```cmd
-  make bundle-publish DIR=./templates/workspaces/base
-  make bundle-register DIR=./templates/workspaces/base BUNDLE_TYPE=workspace
+   make bundle-publish DIR=$AZURETRE_HOME/templates/workspaces/base
```

Comment:
Don't the bundles need to be built first? 
Also we now have commands for doing all 3 at the same time: 

`make workspace_bundle BUNDLE=base`

Reply:
I think we have this in many places in the docs. Worth updating everywhere... Since this PR is already too big, I am thinking on creating an issue to update it across all docs, WDYT?

Reply:
Sounds good